.github/ISSUE_TEMPLATE/question.md

```diff
@@ -1,14 +1,29 @@
 ---
-name: 💬 Questions / Help
-about: If you have questions, please send us an Email
+name: 💬 Questions & Help
```

Comment:
Just for visibility:
https://github.com/cisagov/.github/issues/5#issuecomment-750458857

Reply:
Addressed in https://github.com/cisagov/.github/pull/11/commits/37b275b4f5dbb951b3e408d54fa20e9e0adfa7c1